dataretrieval/nwis.py

```diff
@@ -24,9 +24,12 @@
 WATERDATA_SERVICES = ['qwdata', 'measurements', 'peaks', 'pmcodes', 'water_use', 'ratings']
 
 
-def format_response(df, service=None):
+def format_response(df, multi_index=None, service=None):
     """Setup index for response from query.
     """
+    if multi_index:
+        multi_index=None
```

Comment:
These two lines guarantee multi_Index will be None, it doesn't matter what value.